### PR TITLE
Make logo smaller, clean up formatting, add community tools

### PR DESCRIPTION
**Pull request by nrobinson2000**

### CODE REVIEW

**Comment by nrobinson2000** [comment]
Ping @bsatrom

File: README.md [hunk 1]
```diff
@@ -1,21 +1,22 @@
 <p align="center">
   <br>
-  <img width="400" src="./media/awesome-particle.svg" alt="logo of awesome-particle repository">
```
Comment (bsatrom) [inline README.md:3]: 👍

File: README.md [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
+  <img width="300" src="./media/awesome-particle.svg" alt="logo of awesome-particle repository">
   <br>
   <br>
 </p>
 
-## Awesome Particle [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
```
Comment (bsatrom) [inline README.md]: Please remove all of the heading size chanfes cosmetic changes.

File: README.md [hunk 3]
```diff
@@ -1,21 +1,22 @@
...
+> A curated list of awesome things related to [Particle](https://particle.io)
 
 - [Resources](#resources)
   - [Official Resources](#official-resources)
   - [Tools and SDKs](#tools-and-sdks)
+  - [Community Tools and SDKs](#community-tools-and-sdks)
   - [External Resources](#external-resources)
   - [Conferences](#conferences)
   - [Podcasts](#podcasts)
-  - [Docs](#docs)
```
Comment (bsatrom) [inline README.md]: 👍

File: README.md [hunk 4]
```diff
@@ -71,7 +75,7 @@
 - [Jonathan Beri on Embedded.fm Episode 222](https://www.embedded.fm/episodes/222)
 - [Brandon Satrom on the MacroFab Engineering Podcast Episode 122](https://macrofab.com/blog/mep-ep122-brandon-satrom-lanot/)
 
-## Useful Docs & FAQs
```
Comment (bsatrom) [inline README.md:74]: I don't want to create categories of one, so let's move this under "Useful Docs & FAQs" and change the heading to "Useful Tools, Docs & FAQs"

**Review by bsatrom — CHANGES_REQUESTED** [review]
Thanks for the PR! please make some changes based on my notes below and I'll reconsider merging.

File: README.md (hunk 4 above)
Comment (nrobinson2000) [reply README.md:74]: Are you saying I should put po-util under "Useful Tools, Docs & FAQs" and remove the "Community Tools and SDKs" ?

Comment (bsatrom) [reply README.md:74]: Yes

**Comment by nrobinson2000** [comment]
I think I’ve made all the changes. If there’s any more please notify me.

**Review by bsatrom — APPROVED** [review]

**Pull request merged**